2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -54,13 +75,26 @@ public void Enable_Chassis(boolean enable){
    */
   @Override
   public void robotInit() {
-    imu = new IMU();
-    oi = new OperatorInterface();
+    driveTrainTab = Shuffleboard.getTab("Drive Train"); //The Shuffleboard Tab for all Drive Train related stuff
+
+    m_chooser.setDefaultOption("Default Auto", kDefaultAuto);
+    m_chooser.addOption("My Auto", kCustomAuto);
+    SmartDashboard.putData("Auto choices", m_chooser);
+
     Logger.configureLoggingAndConfig(this, false);
-    
-    
-    
-    
+
+    driveTrain = new Chassis();
+
+    oi = new OperatorInterface();
+
+    m_driveChooser.setDefaultOption("Tank Drive",kTankDrive); //A Drive Train option
+    m_driveChooser.addOption("Arcade Drive", kArcadeDrive); //A Drive Train option
+    m_driveChooser.addOption("Curvature Drive", kCurvatureDrive); //A Drive Train option
+    m_driveChooser.addOption("Shuffle Drive Individual", kShuffleDrive); //A Drive Train option
+    m_driveChooser.addOption("Shuffle Drive Control Groups", kShuffleDriveGroups); //A Drive Train option
+    driveTrainTab.add("Drive Train Choices", m_driveChooser); //Allows you to pick a Drive Train option through Shuffleboard
```

Comment:
I wonder whether this drive train specific dashboard code could be moved into the Chassis class?